lib/templates/erb/scaffold/_form.html.erb

```diff
@@ -0,0 +1,13 @@
+<%%= simple_form_for(@<%= singular_table_name %>) do |f| %>
+  <%%= f.error_notification %>
+
+  <div class="form-inputs">
+  <%- attributes.each do |attribute| -%>
+    <%%= f.<%= attribute.reference? ? :association : :input %> :<%= attribute.name %> %>
```

Comment:
I'll just delete the file. It came with simple_form, and since I'm not using scaffolds, I don't think I'll need it anyway.